lib/util.js

```diff
@@ -66,9 +66,11 @@ const Util = module.exports = {
       // if there are conflicts, log debug output
       if (conflicts.length > 0) {
         _.each(conflicts, conflictKey => {
-          app.log.debug('The config setting', `${key}.${conflictKey}`, 'is defined by both',
-            packName, 'and the application. The', conflictKey,
-            'config defined in', packName, 'will be ignored.')
+          if (app.config[key] && _.isArray(app.config[key][conflictKey])){
```

Comment:
No problem to put it on silly but the sentence have to change for object because object are merged and not ignored. I can modify this PR to do this instead ? And for object I set  `'The config setting', `${key}.${conflictKey}`, 'is defined by both', packName, 'and the application. The', conflictKey, 'config defined in', packName, 'will be ignored.'`